### PR TITLE
Added support for Django 2.1

### PR DESCRIPTION
**Pull request by vThaian**
@yakky could you have a look at my PR?

### CODE REVIEW

File: app_data/fields.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def __init__(self, *args, **kwargs):
         kwargs.setdefault('editable', False)
         super(AppDataField, self).__init__(*args, **kwargs)
 
-    def contribute_to_class(self, cls, name):
+    def contribute_to_class(self, cls, name, **kwargs):
         super(AppDataField, self).contribute_to_class(cls, name)
```
Comment (yakky) [inline app_data/fields.py]: Probably we should propagate `kwargs` to the super call

File: app_data/admin.py [hunk 2]
```diff
@@ -81,15 +81,16 @@ def _get_form_factory_opts(self, request, obj=None, **kwargs):
 
 
 class AppDataModelAdmin(AppDataAdminMixin, ModelAdmin):
-    def get_form(self, request, obj=None, **kwargs):
+    def get_form(self, request, obj=None, change=False, **kwargs):
```
Comment (yakky) [inline app_data/admin.py]: isn't the generic `**kwargs` enough here, given that we pass this unchanged to the super

**Comment by vThaian** [comment]
Thanks, @yakky for your review.

There is a chance to release a new version this week?

**Comment by yakky** [comment]
@vThaian I'll try to have a release ready by tomorrow
merging for now

**Pull request merged**